lua/telescope/actions/init.lua

```diff
@@ -652,6 +652,55 @@ actions.smart_add_to_loclist = function(prompt_bufnr)
   smart_send(prompt_bufnr, 'a', 'loclist')
 end
 
+actions.hop = function(prompt_bufnr, opts)
+  opts = opts or {}
+  local escape = 27
+  local default_keys = {
+    "a", "s", "d", "f", "g", "h", "j", "k", "l", ";",
+    "q", "w", "e", "r", "t", "u", "v", "w",
```

Comment:
w is mentioned twice.

Any reason to not include the whole alphabet?

```lua
for i = 97, 122 do -- a -> z
  print(i, vim.fn.nr2char(i))
end
```

Idk i am not in the hop.nvim world, idk what people do here :laughing:

Reply:
> ```lua
> for i = 97, 122 do -- a -> z
>   print(i, vim.fn.nr2char(i))
> end
> ```

This is black magic conni 😆 , did you sell your soul for it 🤣 🤣 🤣

Reply:
I noticed I repeated some entries locally, but haven't converted the changes yet :sweat_smile: I think a manual configuration is preferred, since the table aligns with the home row. At the same time, I can imagine some users just like 1, 2, .., N better as that might more easily with the mental imagine of what constitutes an entry row.

In any case, I'll make it configurable :) and I think it might be cool to include an optional alternating highlighting of entry rows to even better visualize where the user would jump to

Reply:
yep makes sense. I said i am not a hop person :laughing: